SPECS/filesystem/filesystem.spec

```
@@ -1,715 +1,983 @@
-cat > %{buildroot}/etc/modprobe.d/hfs.conf <<- "EOF"
-# Begin /etc/modprobe.d/hfs.conf
+%files -f filelist
+%defattr(0755,root,root,0755)
+%dir %attr(555,root,root) /
+/bin
+%attr(555,root,root) /boot
+%attr(555,root,root) /afs
+/dev
+%dir /etc
+/etc/X11
+/etc/xdg
+/etc/opt
+/etc/pm
+/etc/skel
+/etc/sysconfig
+/etc/keys
+/etc/pki
+/etc/bash_completion.d/
+%dir /etc/rwtab.d
+%dir /etc/statetab.d
+/home
+/lib
+%ifarch x86_64 ppc64 sparc64 s390x aarch64 ppc64le mips64 mips64el riscv64
+/%{_lib}
+%endif
+/media
+%dir /mnt
+%dir /opt
+%ghost %attr(555,root,root) /proc
+%attr(550,root,root) /root
+/run
+/sbin
+/srv
+%ghost %attr(555,root,root) /sys
+%attr(1777,root,root) /tmp
+%dir /usr
+%attr(555,root,root) /usr/bin
+/usr/games
+/usr/include
+%dir %attr(555,root,root) /usr/lib
+%dir /usr/lib/sysimage
+%dir /usr/lib/systemd
+/usr/lib/systemd/system
+/usr/lib/systemd/user
+%dir /usr/lib/sysusers.d
+%dir /usr/lib/tmpfiles.d
+%dir /usr/lib/locale
+%dir /usr/lib/modules
+%dir /usr/lib/debug
+%dir /usr/lib/debug/.dwz
+%ghost /usr/lib/debug/bin
+%ghost /usr/lib/debug/lib
+%ghost /usr/lib/debug/%{_lib}
+%ghost %dir /usr/lib/debug/usr
+%ghost /usr/lib/debug/usr/bin
+%ghost /usr/lib/debug/usr/sbin
+%ghost /usr/lib/debug/usr/lib
+%ghost /usr/lib/debug/usr/%{_lib}
+%ghost /usr/lib/debug/usr/.dwz
+%ghost /usr/lib/debug/sbin
+%attr(555,root,root) /usr/lib/games
+%ifarch x86_64 ppc64 sparc64 s390x aarch64 ppc64le mips64 mips64el riscv64
+%attr(555,root,root) /usr/%{_lib}
+%else
+%attr(555,root,root) /usr/lib/bpf
+%attr(555,root,root) /usr/lib/X11
+%attr(555,root,root) /usr/lib/pm-utils
+%endif
+/usr/libexec
+/usr/local
+%attr(555,root,root) /usr/sbin
+%dir /usr/share
+/usr/share/aclocal
+/usr/share/appdata
+/usr/share/applications
+/usr/share/augeas
+/usr/share/backgrounds
+%dir /usr/share/bash-completion
+/usr/share/bash-completion/completions
+/usr/share/bash-completion/helpers
+/usr/share/desktop-directories
+/usr/share/dict
+/usr/share/doc
+%attr(555,root,root) %dir /usr/share/empty
+/usr/share/fish
+/usr/share/games
+/usr/share/gnome
+/usr/share/help
+/usr/share/icons
+/usr/share/idl
+/usr/share/info
+%dir /usr/share/licenses
+%dir /usr/share/locale
+%dir /usr/share/man
+/usr/share/metainfo
+/usr/share/mime-info
+/usr/share/misc
+/usr/share/omf
+/usr/share/pixmaps
+/usr/share/sounds
+/usr/share/themes
+/usr/share/xsessions
+%dir /usr/share/X11
+/usr/share/X11/fonts
+/usr/share/wayland-sessions
+/usr/share/zsh
+/usr/src
+/usr/tmp
+%dir /var
+/var/adm
+%dir /var/cache
+/var/cache/bpf
+/var/db
+/var/empty
+/var/ftp
+/var/games
+/var/lib
+/var/local
+%ghost /var/lock
+/var/log
+/var/mail
+/var/nis
+/var/opt
+/var/preserve
+%ghost /var/run
+%dir /var/spool
+%attr(755,root,root) /var/spool/lpd
+%attr(775,root,mail) /var/spool/mail
+%attr(1777,root,root) /var/tmp
+/var/yp
 
-install hfs /bin/true
+%changelog
+* Fri Mar 08 2024 Dan Streetman <ddstreet@microsoft.com> - 3.18-9
+- Initial CBL-Mariner import from Fedora 40 (license: MIT).
```

This has been recently updated and should work now:

```suggestion
- Initial Azure Linux import from Fedora 40 (license: MIT).
```

---

ok